beater/handlers.go

```diff
@@ -76,6 +78,16 @@ func logHandler(h http.Handler) http.Handler {
 	})
 }
 
+func frontendSwitchHandler(feSwitch bool, h http.Handler) http.Handler {
```

Comment:
How about naming this more general like `checkEnabledHandler`? Everything it does is check and react on a boolean in a very generic way.
If you rename it this could be reused for also disabling backend in the future if one only wants to run frontend..